spot-client/src/common/app-state/remote-control-service/actions.js

```diff
@@ -92,7 +182,16 @@ export function goToMeeting(meetingName, options) {
  * @returns {Function}
  */
 export function hangUp(skipFeedback) {
-    return () => remoteControlService.hangUp(skipFeedback);
+    return dispatch => {
+        remoteControlService
+            .hangUp(skipFeedback)
+            .then(() => {
+                dispatch({
+                    type: HANG_UP,
+                    skipFeedback
+                });
+            });
```

Comment:
Why do this dispatch in the promise success but in the other actions the dispatch happens outside the promise chain?